phpunit.xml

```diff
@@ -13,15 +13,13 @@
             <directory>tests/</directory>
         </testsuite>
     </testsuites>
-
-    <logging>
-        <log type="coverage-html" target="tests/coverage/"/>
-    </logging>
-
     <filter>
         <whitelist>
             <directory suffix=".php">src/</directory>
         </whitelist>
+        <blacklist>
```

Comment:
What about the removed coveralls? Took that from #55 our original discussion :+1: